packages/client/src/i18n/messages/form.ts

```diff
@@ -222,6 +222,16 @@ export const formMessageDescriptors = {
     description: 'Label for father select option to certify record',
     id: 'form.field.label.app.certifyRecordTo.father'
   },
+  certifyRecordToGroom: {
+    defaultMessage: 'Print and issue to groom',
+    description: 'Label for groom select option to certify record',
+    id: 'form.field.label.app.certifyRecordTo.groom'
+  },
+  certifyRecordToBride: {
+    defaultMessage: 'Print and issue to bride',
+    description: 'Label for bride select option to certify record',
+    id: 'form.field.label.app.certifyRecordTo.bride'
+  },
```

Comment:
Has the corresponding countryconfig changes been cherry-picked?

Reply:
https://github.com/opencrvs/opencrvs-countryconfig/blob/release-v1.5.0/src/translations/client.csv?plain=1#L914-L918

Yes, its has been.